### DISCUSSION TITLE
Collection runner introduced in v10 doesn't work for scratchpad

### DISCUSSION THREAD

**Discussion by anooprana4236** [Kong/insomnia#8009] [Help] [answered]
I am using collection runner capability introduced in v10 in scratchpad , when i click the play button for scratchpad and select the requests I want to run and run them, it runs the iteration but keeps on showing tests run as 0/0. I can see the requests listed in the iteration pane and when I go to console i can see the requests were run successfully and response was received for all requests. Still the test run shows 0/0 with nothing in passed failed skipped tabs
![image](https://github.com/user-attachments/assets/10915eb6-9d1d-42b3-a581-b23be58a7ff4)

**Comment by ihexxa** [comment, marked as answer]
Hi @anooprana4236 ,
Please try to add one or more tests in the pre-request or after-response script, if it still doesn't work, please let us know. You could also find some examples here: https://docs.insomnia.rest/insomnia/after-response-script
 
<img width="673" alt="image" src="https://github.com/user-attachments/assets/3206a666-94bd-4517-85f9-ce4fb056bf3f">

**Reply by anooprana4236** [reply]
@ihexxa great thanks! This worked fine, after adding post script for test validation I am able to see the corresponding test results once I run the collection using collection runner
